src/main/java/gui/MenuController.java

```diff
@@ -251,33 +248,16 @@ public void clickMouse(MouseEvent mouseEvent) {
      */
     @FXML
     public void traverseGraphClicked() {
-        if (!nodeTextField.getText().equals("") && !radiusTextField.getText().equals("")) {
+        if (!nodeTextField.getText().equals("")) {
             int centreNodeID = Integer.parseInt(nodeTextField.getText());
-            int radius = Integer.parseInt(radiusTextField.getText());
-            zoomController.traverseGraphClicked(centreNodeID, radius);
+            ZoomController.getInstance().traverseGraphClicked(centreNodeID, GraphDrawer.getInstance().getZoomLevel());
             SequenceNode node = GraphDrawer.getInstance().getGraph().getNode(centreNodeID);
             String sequence = DrawableCanvas.getInstance().getParser().getSequenceHashMap().get((long) centreNodeID);
-            infoController.updateSeqLabel(node.toString(sequence));
+            sequenceInfo.setText(node.toString(sequence));
```

Comment:
yup i ditched the motherf*cker, it was redundant because it only updates labels which is easier in menucontroller